CHANGES.rst

```diff
@@ -6,8 +6,10 @@ Changelog
 ------------------
 
 - Rename kitconcept.volto to plone.volto.
-  [tisto] 
+  [tisto]
 
+- Backport "More agnostic and simplify GS profile for Plone Site definition" (https://github.com/kitconcept/kitconcept.volto/pull/38)
```

Comment:
@sneridagh we need to think about how to get the best transition from the kitconcept.volto changelog to the plone.volto changelog. Let's talk about this on Monday. :)